guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -73,40 +73,11 @@ Reboot these hosts after the upgrade has completed.
 # rm /etc/yum.repos.d/*
 ----
 
-. Obtain the latest ISO files by following the {InstallingServerDisconnectedDocURL}downloading-the-binary-dvd-images_satellite[Downloading the Binary DVD Images] procedure in _{InstallingServerDisconnectedDocTitle}_.
+. Create a syncable format export for the {Project} tools and {Project} maintenance repositories. 
+For a list of additional repositories, see xref:synchronizing_the_new_repositories_{context}[].
 
-. Create directories to serve as a mount point, mount the ISO images, and configure the `rhel8` repository by following the {InstallingServerDisconnectedDocURL}configuring-the-base-operating-system-with-offline-repositories_satellite[Configuring the base operating system with offline repositories] procedure in _{InstallingServerDisconnectedDocTitle}_.
-+
-Do not install or update any packages at this stage.
-
-. Configure the {Project} {ProjectVersion} repository from the ISO file.
-
-.. Copy the ISO file's repository data file for the {ProjectName} packages:
-+
-[options="nowrap"]
-----
-# cp /media/sat6/Satellite/media.repo /etc/yum.repos.d/satellite.repo
-----
-
-.. Edit the `/etc/yum.repos.d/satellite.repo` file:
-+
-----
-# vi /etc/yum.repos.d/satellite.repo
-----
-
-... Change the default `InstallMedia` repository name to `{Project}-{ProjectVersion}`:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-[{Project}-{ProjectVersion}]
-----
-
-... Add the `baseurl` directive:
-+
-[options="nowrap"]
-----
-baseurl=file:///media/sat6/Satellite
-----
+. Configure the {Project} {ProjectVersion} repository from the syncable format export.
```

Comment:
I peeked into the Jira and noticed that this might actually be supposed to replace the subsequent step. Can you please double check the Jira ticket?

Reply:
You are right!. After removing steps 9 and 10, I think the numbering threw me off. Thank you!

Reply:
Actually I think the reporter confused Step 11 with Step 10 and meant to say Step 10 in the ticket. This is good as is.